index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title id="title">Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="css/styles.css">
+      </head>
+
+  <body>
+    <main id="main">
+      <h1><center>Dr. Norman Borlaug</center></h1>
+      <p><center>The man who saved a billion lives</center></p>
+      <div id="img-div">
+        <center><img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" ></center>
```

Comment:
I think it would be better to center your documents using styles.css than HTML.

Reply:
- [ ] As suggested by the previous reviewer, kindly make sure you the reason "Why" you suggest the changes in question as it's an occasion for the student to learn.

Reply:
- [ ] As suggested by the previous reviewer, kindly make sure you the reason "Why" you suggest the changes in question as it's an occasion for the student to learn.